Assets/Scripts/UI/PickupPup/MainMenu/MainMenu.cs

```diff
@@ -6,15 +6,20 @@
 
 using UnityEngine;
 using UnityEngine.UI;
+using UnityEngine.Advertisements;
```

Comment:
I think ideally we should have a separate class to handle advertisements, we shouldn't have the logic stored inside of `MainMenu`